.circleci/config.yml

```diff
@@ -0,0 +1,49 @@
+version: 2.1
+jobs:
+  unit:
+    docker:
+      - image: circleci/node:8
+    steps:
+      - run: node --version
+      - run: npm --version
```

Comment:
Do we need `node --version` and `npm --version` here ?

Reply:
good q. I took this from a ci setup we had in another proj (maybe cli?). Fine to remove it if you also dont think it's necessary